src/widgets/WebexMeetingWidget.css

```diff
@@ -0,0 +1,7 @@
+.meeting-widget {
+  display: flex;
+  justify-content: center;
```

Comment:
Is there a reason for centering the contents?

Reply:
yes, the spinner that you asked to be rendered needs to be centered.